tests/dags/test_descendant_dags.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
```

Comment:
I prefer including it with the test it makes it more clear what is being tested and will not add the time consumed by the backfill tests.